packages/trie/src/util/walkController.ts

```diff
@@ -66,7 +66,7 @@ export class WalkController {
    * @param node - Node to get all children of and call onNode on.
    * @param key - The current `key` which would yield the `node` when trying to get this node with a `get` operation.
    */
-  allChildren(node: TrieNode, key: Nibbles = []) {
+  allChildren(node: TrieNode, progress: Nibbles = []) {
```

Comment:
:thumbsup

i like this name change. we should adopt it elsewhere in trie methods.

Reply:
Yes, I have the impression that especially in the trie library these kind of simple and local changes can add up to a significantly better readability if we apply consistently.